core/src/main/scala/com/softwaremill/react/kafka2/Consumer.scala

```diff
@@ -46,11 +46,10 @@ object ManualCommitConsumer {
   type CommitConfirmation = Future[CommitMsg]
 
   case class ConsumerShape[K, V](
-    commit: Inlet[CommitMsg],
-    messages: Outlet[ConsumerRecord[K, V]],
-    confirmation: Outlet[CommitConfirmation]
-  ) extends Shape
-  {
+      commit: Inlet[CommitMsg],
```

Comment:
Could you please reformat only changed code? Such reformats are really confusing